vital/algo/keyframe_selection.h

```diff
@@ -36,17 +34,14 @@ class VITAL_ALGO_EXPORT keyframe_selection
   static std::string static_type_name() { return "keyframe_selection"; }
 
   /// Select keyframes from a set of tracks.
-
-  /** Different implementations can select key-frames in different ways.
-   *   For example, one method could only add key-frames for frames that are
-   * new.  Another could increase the
-   * density of key-frames near existing frames so dense processing can be
-   * done.
-   */
-  /**
-   * \param [in] tracks The tracks over which to select key-frames
-   * \returns a track set that includes the selected keyframe data structure
-   */
+  ///
+  /// Different implementations can select key-frames in different ways.
+  ///   For example, one method could only add key-frames for frames that are
+  /// new.  Another could increase the
+  /// density of key-frames near existing frames so dense processing can be
+  /// done.
+  /// \param [in] tracks The tracks over which to select key-frames
```

Comment:
It's possible there should be an empty line in between here.

Reply:
In this PR there were 101 files processed. I reviewed all of them and found 2 problems.  Daniel spotted this one that I missed.  So that make 3 issues in 101 files.  Of those, this is the only one of this type, so I think they are rare.  I would think it would be simple to find and eradicate them.

Reply:
Okay, it looks like this it the only "blocker" in *this* PR. TBH we could probably repair this "by hand" and go ahead and merge this, but it might be safer to fix the script so these won't be able to slip through when we run this on the rest of the code base.

I'm mildly concerned about non-documentation comments also being changed, but I didn't see any that weren't include guards. For now I don't mind changing those, but it might similarly be a concern for running this on the rest of the code base.

Reply:
I've updated the script with a few more regex-based replacements.  A new PR will be coming shortly.